lib/src/signing.rs

```diff
@@ -71,8 +87,14 @@ impl Verification {
             status,
             key,
             display,
+            backend: None,
         }
     }
+
+    /// The name of the backend that provided this verification.
+    pub fn backend(&self) -> Option<&str> {
+        self.backend.as_deref()
+    }
```

Comment:
nit: this method is no longer needed since `backend` is public.

Oh, btw, the `backend` field is unused right now. Do you remember where it will be used?

Reply:
oh, actually

I think it should be in the templater?.
Like `commit.signature().backend()`
That's the only thing I recall about the backend field, maybe there was something else..
Anyway it's definitely not a core part/dealbreaker, so if it's unused and we dont need it in the templater we can get rid of it then

Reply:
Oh true, in #3141 we had `signature.backend()` in the templater!
I must have lost that.. I will bring it back. Thanks for spotting that!